ddev/tests/cli/release/branch/test_tag.py

```diff
@@ -0,0 +1,191 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from unittest.mock import call as c
+
+import pytest
+
+from ddev.utils.git import GitRepository
+
+NO_CONFIRMATION_SO_ABORT = 'Did not get confirmation, aborting. Did not create or push the tag.'
+
+
+@pytest.fixture
+def basic_git(mocker):
+    mock_git = mocker.create_autospec(GitRepository)
+    # We're patching the constructor (__new__) method of GitRepository class.
+    # That's why we need a function that returns the mock.
+    mocker.patch('ddev.repo.core.GitRepository', lambda _: mock_git)
+    return mock_git
+
+
+@pytest.fixture
+def git(basic_git):
+    basic_git.current_branch.return_value = '7.56.x'
+    basic_git.tags.return_value = [
+        # interesting phenomena:
+        # - skipping RCs
+        # - rc 11 naively would be sorted as less than rc 2
+        # - an rc tag from dbm
+        # '7.56.0',
```

Comment:
Maybe this should be reworded, I don't quite follow

Reply:
👍 I'll mark each one.